### ISSUE TITLE
Spinner - DS site WCAG accessibility documentation

### ISSUE THREAD

**Issue by jcfilben** [grommet/hpe-design-system#5018]
Add Accessibility information to the Spinner page on the Design System site.

- [x] Assess which [WCAG rules](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.2) are applicable to the component and if they are passing/failing.
- [x] Create a pull request on the Design System site to add this information to the component page. Follow the format here: https://github.com/grommet/hpe-design-system/pull/5004/files. The finished result should look something like this: https://design-system.hpe.design/components/menu?q=menu#wcag-compliance.
- [x] Create issues in the relevant repository (grommet, grommet-theme-hpe, etc.) for any rules where we are not meeting WCAG AA or if there is something observed that it feels like we should fix. We should create 1 issue per rule failure. Issues where we are not meeting AA should be tagged with 'bug'. All issues created through this process should have the 'accessibility' label and be added to the accessibility board.

The pull request should be up on Github within the first week of the sprint so that the second week of the sprint can be focused on review and polishing.

When your work is ready for review please get a first pass review from another member of the accessibility sprint and once that is done reach out to Jessica to flag that a final review is needed.

Resources and videos for this work can be found here: https://hpe-my.sharepoint.com/shared?id=%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents%2FHPE%20Design%20System%2FAccessibility&listurl=https%3A%2F%2Fhpe%2Esharepoint%2Ecom%2Fteams%2Fhpe%2Ddesign%2FShared%20Documents

**Comment by luketa8** [comment]
Added first review to [Spreadsheet](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/_layouts/15/Doc.aspx?sourcedoc=%7BE975E19F-3264-4F4C-BD0A-446F23F89277%7D&file=WCAG%20Rule%20Evaluation%20Template.xlsx&action=default&mobileredirect=true&wdOrigin=OUTLOOK-METAOS.FILEBROWSER.FILES-HOME)

**Comment by oliverHPE** [comment]
Thanks @luketa8 Have amended according to comments. Still unsure criterion 2.2.2 . @jcfilben Think It's ready to be handed over to you.

**Comment by jcfilben** [comment]
@oliverHPE I've gone through and added notes in the [excel file](https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=mDJjP3&nav=MTVfezM2QThDQzlELTA2MzMtNDZBNi1CRjM5LTdFNTI2NjUwNDM3Mn0)

**Comment by oliverHPE** [comment]
@jcfilben Pull request done.

**Comment by britt6612** [comment]
@oliverHPE PR was merged did you add an issue for the rule 1.4.11 Non-text Contrast that Spinner is currently failing?

**Comment by britt6612** [comment]
great thanks @taysea will close this issue then
